Cargo.lock

```diff
@@ -150,7 +150,7 @@ dependencies = [
 [[package]]
 name = "atk"
 version = "0.13.0"
-source = "git+https://github.com/gtk-rs/gtk-rs#18bafda64a6d8cd5b4e23c2053543853f75ed7b7"
```

Comment:
We need to avoid changing the version of anything related to gstreamer, gtk, and glib. These often have breaking changes. When adding dependencies to Cargo.toml, it will want to update to the latest version. You'll then need to manually edit Cargo.lock and search/replace commit hashes to undo the changes.

Reply:
revert back